.jenkins/Jenkinsfile

```diff
@@ -16,524 +16,486 @@
-                warnError('missing test xml files') {
-                    script {
-                        copyTestResults('test-cdc', cdc.getNumber())
-                    }
-                }
-            }
+       stage('stress') {
+         steps {
+           testJob(".build/docker/run-tests.sh", "stress-test", 1)
+         }
+       }
+       stage('fqltool') {
+         steps {
+           testJob(".build/docker/run-tests.sh", "fqltool-test", 1)
+         }
+       }
+       stage('units') {
+         steps {
+           testJob(".build/docker/run-tests.sh", "test", 8)
+         }
+       }
+       stage('long units') {
+         steps {
+           testJob(".build/docker/run-tests.sh", "long-test", 8)
+         }
+       }
+       stage('burn') {
+         steps {
+           testJob(".build/docker/run-tests.sh", "test-burn", 8)
+         }
+       }
+       stage('cdc') {
+         steps {
+           testJob(".build/docker/run-tests.sh", "test-cdc", 8)
+         }
+       }
+       stage('compression') {
+         steps {
+           testJob(".build/docker/run-tests.sh", "test-compression", 8)
+         }
+       }
+       stage('cqlsh') {
+         steps {
+           testJob(".build/docker/run-tests.sh", "cqlsh-test", 1)
+         }
+       }
+     }
+    }
+    // stage('Distributed Test') {
+    //  parallel {
+    //    stage('jvm-dtest') {
+    //      steps {
+    //        testJob(".build/docker/run-tests.sh", "jvm-dtest", 8)
+    //      }
+    //    }
+    //    stage('jvm-dtest-upgrade') {
+    //      steps {
+    //        testJob(".build/docker/run-tests.sh", "jvm-dtest-upgrade", 8)
+    //      }
+    //    }
+    //    stage('dtest') {
+    //      steps {
+    //        testJob(".build/docker/run-tests.sh", "dtest", 64)
+    //      }
+    //    }
+    //    stage('dtest-large') {
+    //      steps {
+    //        testJob(".build/docker/run-tests.sh", "dtest-large", 8)
+    //      }
+    //    }
+    //    stage('dtest-novnode') {
+    //      steps {
+    //        testJob(".build/docker/run-tests.sh", "dtest-novnode", 64)
+    //      }
+    //    }
+    //    stage('dtest-offheap') {
+    //      steps {
+    //        testJob(".build/docker/run-tests.sh", "dtest-offheap", 64)
+    //      }
+    //    }
+    //    stage('dtest-large-novnode') {
+    //      steps {
+    //        testJob(".build/docker/run-tests.sh", "dtest-large-novnode", 8)
+    //      }
+    //    }
+    //    stage('dtest-upgrade') {
+    //      steps {
+    //        testJob(".build/docker/run-tests.sh", "dtest-upgrade", 64)
+    //      }
+    //    }
+    //  }
+    // }
+    stage('Summary') {
+      steps {
+       // FIXME is this needed if each job is doing it ??
+       //junit testResults: '**/build/test/**/TEST*.xml,**/cqlshlib.xml,**/nosetests.xml', testDataPublishers: [[$class: 'StabilityTestDataPublisher']]
+
+       //sendNotifications()
+       
+       // FIXME how to collect and build one test report file ?? 
```

Comment:
So we have this method called below but it looks like it's doing this for a given specific test axis. Is the outstanding question "how do we do this for multiple axes generated by the testJob"?